.travis.yml

```diff
@@ -60,17 +63,16 @@ matrix:
   include:
     - os: osx
       env: BUILD_TYPE=Release BUILD_EXTERNAL=0 MAKE_TARGETS="llvm-spirv" MAKE_TEST_TARGET="check-llvm-spirv"
-      osx_image: xcode9.3
+      osx_image: xcode12
```

Comment:
Actually I noticed the xcode jobs are timing out after 50m in your travis run, whereas they used to take about 30m.

Reply:
Right, quite strange.
I will compare the 2 travis run.